src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1114,7 +1114,7 @@ public void RescheduleNextIdleDatabaseActivity(string databaseName, IdleDatabase
             if (idleDatabaseActivity == null)
             {
                 if (_wakeupTimers.TryRemove(databaseName, out var oldTimer))
-                    oldTimer.Dispose();
+                    oldTimer.Value.Dispose();
```

Comment:
do we want to check if it was materialized?

Reply:
done